compiler/src/dotty/tools/dotc/core/tasty/TreePickler.scala

```diff
@@ -426,6 +426,9 @@ class TreePickler(pickler: TastyPickler) {
         case CaseDef(pat, guard, rhs) =>
           writeByte(CASEDEF)
           withLength { pickleTree(pat); pickleTree(rhs); pickleTreeUnlessEmpty(guard) }
+        case Labeled(bind, expr) =>
+          writeByte(LABELED)
+          withLength { pickleTree(bind); pickleTree(expr) }
```

Comment:
Right now we will never pickle `Labeled` trees as they are generated in pattern matcher. We should add this code only when we will need it for `while` and `do while`